docs/install.md

```diff
@@ -1302,6 +1314,140 @@ deployment is in the `Failed` state, then one can look into the event(s) using
 > [`DeploymentSpec`](#deployment-crd-api).
 
 
+## PMEM-CSI operator API conversion webhook
+
+The operator release v0.9.0 deprecates the `v1alpha1` [deployment API](#pmem-csi-deployment-crd)
+in favor of `v1beta1`. To keep supporting a custom resource that uses the
+`v1alpha1` version must be converted to the latest supported `v1beta1` CRD
+version. This conversion is done by the Kubernetes API server by using the
+conversion strategy chosen in the [deployment CRD sepcification](#pmem-csi-deployment-crd).
+If no conversion strategy is chosen by the CRD, by default the API server
+converts the custom resource `apiVersion` to the latest supported CRD version.
```

Comment:
Converts when? When a client asks for v1beta1 (i.e. on demand)?

Reply:
> Converts when? When a client asks for v1beta1 (i.e. on demand)?

As pet the [webhook conversion](https://v1-15.docs.kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/#webhook-conversion) documentation:

```
The API server also supports webhook conversions that call an external service in case a conversion is required. For example when:

- custom resource is requested in a different version than stored version.
- Watch is created in one version but the changed object is stored in another version.
- custom resource PUT request is in a different version than storage version.
```